pkgs/applications/networking/instant-messengers/rambox/default.nix

```diff
@@ -32,6 +42,9 @@ stdenv.mkDerivation rec {
     mkdir -p $out/bin $out/share/rambox
     cp -r * $out/share/rambox
     ln -s $out/share/rambox/Rambox $out/bin
+
+    cp -r ${desktopItem}/share/applications $out/share
+    install -Dm644 ${./rambox.png} $out/share/pixmaps/rambox.png
```

Comment:
sorry, but we should not merge binary files into nixpkgs, because it slows down git, which is nasty for a project of this scale. You can fetch the icon also from here: https://raw.githubusercontent.com/saenzramiro/rambox/9e4444e6297dd35743b79fe23f8d451a104028d5/resources/Icon.png